src/utils/DocumentElementHelper.js

```diff
@@ -0,0 +1,24 @@
+import HTMLElement from '../HTMLElement';
+
+/**
+ * Class provide processing for innerHTML function for Document.documentElement
+ * element. When this element property innerHTML changes, it also updates links
+ * for head, body elements in document.
+ */
+export default class DocumentElementHelper extends HTMLElement {
```

Comment:
Well it's not real element. It just provides interface for documentElement field. See https://developer.mozilla.org/en-US/docs/Web/API/Document/documentElement

After your comment i'm thinking of making just getter for that field in Document class and provide object with reference to documents body, head etc.